modules/telemetry/events/settings-events.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace VIPWorkflow\Modules\Telemetry\Events;
+
+use VIPWorkflow\Modules\Telemetry\Tracker;
+use VIPWorkflow\VIP_Workflow;
+
+function record_admin_update( string $previous_version, string $new_version, Tracker $tracker ): void {
+	// Get all custom statuses
+	$custom_statuses = VIP_Workflow::instance()->custom_status->get_custom_statuses();
+
+	// Get all posts count
+	$posts_count = wp_count_posts();
+	// Only care about published and posts with custom status
+	$total_posts = (int) $posts_count->publish;
```

Comment:
I'm curious why published posts' count is summed with custom statuses. It seems like a customer with 50 custom status posts and 50 published posts would have the same event data as a customer with 0 custom status posts and 100 published posts.

Maybe we can send `total_posts` and `custom_status_posts` counts as separate pieces of event data?